msteams-platform/tabs/tabs-link-unfurling.md

```diff
@@ -9,49 +9,73 @@ ms.localizationpriority: high
 
 # Tabs link unfurling and Stage View
 
-Stage View is a new user interface (UI) component. It allows you to render the content that is opened in full screen in Teams and pinned as a tab.
+Stage View is a user interface (UI) component that allows you to render content either a full-sized modal within Teams or as a new window.
```

Comment:
Grammatical mistake. Please check.

Reply:
Stage View is a user interface (UI) component that allows you to render content either as full-sized modal within Teams or a new chat window.